wifiphisher/common/interfaces.py

```diff
@@ -517,6 +551,9 @@ def get_interface(self, has_ap_mode=None, has_monitor_mode=None):
 
         # return interface if found otherwise raise an error
         if chosen_interface:
+            adapter = self._name_to_object[chosen_interface]
+            if adapter.is_managed_by_nm:
```

Comment:
do you mean our algorithm will choose wlan1 but since it is managed so we'll raise the exception here?
maybe we can put the check in the if condition: how do you think?
```python
if interface not in self._active and not adapter.is_managed_by_nm:
```